bitclock-fw/main/libs/gatt.c

```diff
@@ -105,6 +106,12 @@ static const ble_uuid128_t gatt_svr_chr_weather_path_uuid =
 uint16_t gatt_svr_chr_weather_path_val_handle;
 char gatt_svr_chr_weather_path_val[64];
 
+static const ble_uuid128_t gatt_svr_chr_mqtt_url_uuid =
+    BLE_UUID128_INIT(0x2a, 0xb4, 0x8c, 0x15, 0x93, 0x4d, 0x11, 0xec, 0xb9, 0x0a,
```

Comment:
It's annoying but you have to reverse the bytes here.

```
static const ble_uuid128_t gatt_svr_chr_weather_path_uuid =
    BLE_UUID128_INIT(0x81, 0x1f, 0x8c, 0xd4, 0x26, 0x00, 0x80, 0xb8, 0xe2, 0x47,
                     0x66, 0x95, 0x1d, 0xee, 0xef, 0x32);
```
`let CHR_WEATHER_PATH_UUID = "32efee1d-9566-47e2-b880-0026d48c1f81";`

I don't immediately remember why.